src/index.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar__length--31 calendar__day--sun">
```

Comment:
```suggestion
      <div class="calendar calendar__length--31 calendar__startday--sun">
```
this class looks like the whole `div` is sunday.